src/3rdparty/walkontable/src/overlay/table/bottom.js

```diff
@@ -0,0 +1,40 @@
+import OverlayTable from './_base';
+
+class BottomOverlayTable extends OverlayTable {
+  getFirstRenderedRow() {
+    const index = this.wot.getSetting('totalRows') - this.wot.getSetting('fixedRowsBottom');
```

Comment:
I think it would be a good idea to add an empty line between the variable declarations and the method logic, for better visual separation. 

This applies to all other methods with this structure.